spec/mem/pointer.md

```diff
@@ -25,6 +25,10 @@ We therefore use the term *thin pointer* for what has been described above, and
 /// of the address space.
 pub type Address = Int;
 
+/// A "trait name" is an identifier for the trait a vtable is for.
+/// This defines the size and function signatures of the methods.
```

Comment:
```suggestion
/// This defines the number of methods and their function signatures.
```
This isn't about the size of the type, right?

Reply:
Yes, size was ambiguous.